### PR TITLE
Update frontend to fd07152

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

File: supervisor/api/panel/entrypoint.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 
 try {
-  new Function("import('/api/hassio/app/frontend_latest/entrypoint.js')")();
+  new Function("import('/api/hassio/app/frontend_latest//api/hassio/app/frontend_latest/entrypoint.c64d3ee6.js')")();
```
Comment (Misiu) [inline supervisor/api/panel/entrypoint.js:3]: is this correct?
I think it should be:
``` javascript
new Function("import('/api/hassio/app/frontend_latest/entrypoint.c64d3ee6.js')")();
```

File: supervisor/api/panel/entrypoint.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 } catch (err) {
   var el = document.createElement('script');
-  el.src = '/api/hassio/app/frontend_es5/entrypoint.js';
+  el.src = '/api/hassio/app/frontend_es5//api/hassio/app/frontend_es5/entrypoint.4a2722b2.js';
```
Comment (Misiu) [inline supervisor/api/panel/entrypoint.js:6]: same as above - double slash issue.

Comment (ludeeus) [reply supervisor/api/panel/entrypoint.js:6]: double slash is not the issue

**Comment by ludeeus** [comment]
Build scripts needs adjustments

**Pull request closed without merging**